src/ws-date-picker/ws-date-picker.scss

```diff
@@ -2,6 +2,7 @@
   position: relative;
   display: inline-block;
   &.with-input .icon {
+    pointer-events: none;
```

Comment:
I din't know this yet. I absolutely like this!!!! Before I always created additional listeners for the icons^^